src/server/modules/contact/index.js

```diff
@@ -1,6 +0,0 @@
-// Components
-import schema from './schema.graphql';
-import createResolvers from './resolvers';
-import Feature from '../connector';
-
-export default new Feature({ schema, createResolversFunc: createResolvers });
```

Comment:
Looks like Git didn't understand that you have renamed this file, because you have both renamed and changed its contents. In such cases I recommend to rename all files in the first commit and changed them in the second commit. Unfortunately I will be unable to merge this PR if it will have places where Git have lost track of some file and diff has all `-` on all `+`, because it will be a major headache to merge this change into multiple other open PRs

Reply:
I'll retract this PR and try again.